src/routes/dataset/[datasetId]/+page.server.ts

```diff
@@ -1,11 +1,22 @@
 import { api } from '$lib/api.server';
+import { backendUrl } from '$lib/config';
 import type { ServerLoad } from '@sveltejs/kit';
 
 export const load: ServerLoad<{ datasetId: string }> = async ({ params, fetch }) => {
 	const { getSchema } = api(fetch);
 
 	// we have to fetch the jsonld server side only, because of CORS.
 	const jsonld = await getSchema(params.datasetId);
+
+	// Calling the CKAN-dataset-page in the background as this will trigger creating default previews
+	const datasetId = params.datasetId;
+	const datasetBackendUrl = backendUrl(`dataset/${datasetId}`);
+
+	// Fire-and-forget the background call
```

Comment:
the only things I am thinking here it is to do this background call only for valid  datasetId;
so maybe checking if datasetId exist before would be a good idea, wdyt?

Reply:
I like that idea! But: to check whether a dataset-id is valid, I'd have to do a request to CKAN first. So then I would have two requests in the background :/
If the dataset does not exist (or the call produces an error) the error will be swallowed and logged, but nothing more.

Reply:
morst important that we know where it breaks, in case it happens